composer.json

```diff
@@ -1,5 +1,5 @@
 {
-    "name": "nocworx/psr7",
+    "name": "nocworx/guzzlehttp-psr7",
```

Comment:
if you check the original repo, the vendor name is `guzzle` not `guzzlehttp`

Reply:
Will update